extensions/legit/porcelain-git.lisp

```diff
@@ -0,0 +1,416 @@
+(uiop:define-package :lem/porcelain-git
+  (:use :cl :lem/porcelain)
+  (:shadow :push)
+  (:import-from :trivial-types
+                :proper-list)
+  (:import-from :lem/porcelain
+                :porcelain-error)
+  (:export :git-project-p)
+  (:documentation "Implements the porcelain interface for git-based repos."))
+
+(in-package :lem/porcelain-git)
+
+(declaim (type (proper-list string) *git-base-arglist*))
+(defvar *git-base-arglist* (list "git")
```

Comment:
git package + git- prefix = redundant. 

(but again, wait, let's focus on important matters)

Reply:
Resolving this, if/when the time comes to address it I'll just track everything under the equivalent comment on fossil